x-pack/plugins/case/common/api/cases/case.ts

```diff
@@ -20,21 +22,42 @@ const CaseBasicRt = rt.type({
   title: rt.string,
 });
 
+const CasePushBasicRt = rt.type({
```

Comment:
is there a reason some of these are named `CasePush` and others `PushCase`? If not, could you please go with one or the other?

Reply:
The `CasePush` refers to Cases API and `PushCase` to ServiceNow API. I agree that is a bit confusing.